### ISSUE TITLE
[FABRIC] Fuzzy Card on export bus causes crashes when trying to find/match armor to export

### ISSUE THREAD

**Issue by ConanODoyle** [AppliedEnergistics/Applied-Energistics-2#5214]
**Describe the bug**
Fuzzy match export causes crashes when exporting item with match

**To Reproduce**
Set up AE2 network, flood it with various armor drops from zombie mobs (various damage levels, various enchantments)
Set up export bus to export armor with fuzzy matches into a chest.

Matches used in the crash setup is no-damage gold chestplate, 25% damaged gold boots, 40% damaged gold leggings, 40% damaged gold helmet, stone tools, leather armor of various damage levels. Modpack is All of Fabric 3 (April 19th release).

**Additional context**
Full logs coming (have to wait on server host to get them)
Relevant portion they have already screenshotted:
![unknown](https://user-images.githubusercontent.com/10893412/118208389-88f7f400-b41b-11eb-9474-3eef722d66ff.png)

**Environment**
1.16.5, AE2 version 8.2.0-alpha.3

**Comment by Technici4n** [comment]
Duplicate of #4821
